doc/smart-switch/high-availability/smart-switch-ha-detailed-design.md

```diff
@@ -59,138 +86,230 @@ flowchart LR
    subgraph NPU Components
       NPU_SWSS[swss]
       NPU_HAMGRD[hamgrd]
+      NPU_SYNCD[syncd]
 
       subgraph CONFIG DB
-         NPU_DPU[DPU_TABLE]
-         NPU_VDPU[VDPU_TABLE]
-         NPU_DASH_HA_GLOBAL_CONFIG[DASH_HA_GLOBAL_CONFIG]
+         subgraph All NPUs
+            NPU_DPU[DPU]
+            NPU_VDPU[VDPU]
+            NPU_DASH_HA_GLOBAL_CONFIG[DASH_HA_GLOBAL_CONFIG]
+         end
       end
 
       subgraph APPL DB
+         NPU_DASH_ENI_TUNNEL_TABLE[DASH_ENI_TUNNEL_TABLE]
```

Comment:
Please provide clear schema for the DASH_ENI_TUNNEL_TABLE.

Reply:
table added as DASH_ENI_FORWARD_TABLE.